dashboards/traefik/traefik.json

```diff
@@ -138,7 +138,7 @@
             "nrqlQueries": [
               {
                 "accountId": 0,
-                "query": "FROM Metric SELECT\nlatest(traefik_entrypoint_open_connections) AS 'Open Connections',\nsum(traefik_entrypoint_requests_total) AS 'Total Requests',\nsum(traefik_entrypoint_requests_tls_total) AS 'Total TLS Requests',\nbucketPercentile(traefik_entrypoint_request_duration_seconds_bucket, 95) AS 'Request Duration (ms)',\naverage(traefik_entrypoint_request_duration_seconds_sum) AS 'Average Request Duration (ms)'\nFACET prometheus_server, instance, entrypoint, method, protocol\nLIMIT MAX\nWHERE instrumentation.name = 'remote-write'"
+                "query": "FROM Metric SELECT\nlatest(traefik_entrypoint_open_connections) AS 'Open Connections',\nsum(traefik_entrypoint_requests_total) AS 'Total Requests',\nsum(traefik_entrypoint_requests_tls_total) AS 'Total TLS Requests',\nbucketPercentile(traefik_entrypoint_request_duration_seconds_bucket, 95) AS 'Request Duration (ms)',\naverage(traefik_entrypoint_request_duration_seconds_sum) AS 'Average Request Duration (ms)'\nFACET prometheus_server, instance, entrypoint, method, protocol\nLIMIT MAX\nWHERE instrumentation.name = 'remote-write' AND prometheus_server IN ( {{prometheus_server}}) AND instance IN ( {{instance}} ) facet concat(prometheus_server,' - ',instance)"
```

Comment:
2 `FACET` conditions in this query; recommend consolidating into:

```
FACET concat(prometheus_server,' - ',instance) AS 'Instance', entrypoint, method, protocol
```

Reply:
Thanks, changes made.